doc/WebRTC.xml

```diff
@@ -0,0 +1,1022 @@
+        </imageobject>
+      </mediaobject>
+    </figure>
+    <section xml:id="section_ayt_v12_v5b">
+      <title>Connecting to the signaling server</title>
+      <para>Both a client and a device need to connect to the server by setting up a WebSocket with
+        valid access tokens that the server will accept. The connection handling is done using
+        WebSocket [RFC 6455].</para>
+      <para>The signaling messages are then sent as JSON text messages in the WebSocket. Please
+        refer to the Signaling Protocol AsyncAPI schema for specific details regarding the Signaling
+        Protocol.</para>
+      <section xml:id="section_hsq_qhn_rwb">
+        <title>Client</title>
+        <para>A client connects to the server by setting up a WebSocket to, for example:
+          <programlisting>wss://signaling.example.com/client?authorization=Bearer%20access_token</programlisting></para>
+        <para>Where <code>signaling.example.com</code> should be replaced with the server's address,
+          and <code>access_token</code> should be replaced with a valid access token that the server
+          will accept. See section <xref linkend="chapter_cwp_r2k_v5b"/> for how to get an access
+          token.</para>
+        <para>The WebSocket sub protocol shall be set to 'client.webrtc.onvif.org' for the
+          client.</para>
+      </section>
+      <section xml:id="section_isq_qhn_rwb">
+        <title>Device</title>
+        <para>A device must first be onboarded in order to be able to set up a WebSocket to the
+          server. See section <xref linkend="section_nsq_qhn_rwb"/> for more information. An
+          onboarded device then connects by setting up a WebSocket to, for example:</para>
+        <programlisting>wss://signaling.example.com/device?deviceId=myDeviceId&amp;orgId=myOrgId&amp;token=myToken</programlisting>
+        <para>Where <code>signaling.example.com</code> should be replaced with the server's address
+          and the <code>deviceId</code> and <code>orgId</code> (decided when onboarding) are used as
+          query strings when connecting. An access token needs to be provided (<code>token</code>).
+          See section <xref linkend="section_dwp_r2k_v5b"/> for information about how a device gets
+          an access token.</para>
+        <para>The WebSocket sub protocol shall be set to 'device.webrtc.onvif.org' for the
+          device.</para>
+      </section>
+      <section xml:id="section_zdt_ypd_svb">
+        <title>hello</title>
+        <para>Directly after connecting a WebSocket to the server the client or device gets a hello
+          message from the server, with the peer's assigned Signaling ID.</para>
+      </section>
+    </section>
+    <section xml:id="section_byt_v12_v5b">
+      <title>Message structure</title>
+      <para>Generally a message contains two parts: header and payload. The header part is processed
```

Comment:
This is the only part of the spec talking about this, but it is not very clear on how those two parts go together. The way we interpreted this internally for now is as as base JSON `{"header":{...}, "payload":{...}}` but this should be explicit somewhere.